eng/pipelines/templates/Apex_Tests_On_Windows.yml

```diff
@@ -41,11 +41,12 @@ stages:
           - ${{parameters.variables}}
       runSettingsURI: ${{parameters.runSettingsURI}}
       visualStudioBootstrapperURI: $(bootstrapperUrl)      
-      baseVisualStudioBootstrapperURI: ${{parameters.baseBuildDrop}};bootstrappers/Enterprise/vs_enterprise.exe
+      baseVisualStudioBootstrapperURI: https://vsdrop.corp.microsoft.com/file/v1/$(VisualStudio.BaseBuild.ProductsDropName);bootstrappers/Enterprise/vs_enterprise.exe
```

Comment:
We should do the same change [here]( https://github.com/NuGet/NuGet.Client/blob/dev/eng/pipelines/templates/End_To_End_Tests_On_Windows.yml#L55) also.